src/main/scala/akka/contrib/datareplication/Replicator.scala

```diff
@@ -951,7 +960,7 @@ class Replicator(settings: ReplicatorSettings) extends Actor with ActorLogging {
 
     // notify subscribers, later
     if (subscribers.contains(key) && !changed.contains(key)) {
-      val newDigest = if (envelope.data == DeletedData) DeletedDigest else ByteString.fromArray(MessageDigest.getInstance("SHA-1").digest(serializer.toBinary(envelope)))
+      val newDigest = if (envelope.data == DeletedData) DeletedDigest else digest(envelope)
```

Comment:
I am trying to using it to implement an service discovery hub as an instead of the clusterpubsub